ui/package.json

```diff
@@ -4,7 +4,8 @@
   "description": "UI for the Kafka Java Vertx starter application",
   "main": "src/index.js",
   "scripts": {
-    "test": "test"
+    "test": "test",
+    "build": "webpack"
```

Comment:
Looks to me like webpack isnt here (yet) - could this be changed to a no-op type command for now?

Reply:
Ah whoops! I must have a global install of webpack because this was happy for me